wordpress-mobile/WordPress-iOS
# Information Architecture Improvements

Ref #13580, #13319

#### Adds a few improvements and fixes related to the above issues of the [IA project](https://github.com/wordpress-mobile/WordPress-iOS/projects/15)

- Fix reader bar divider colors in dark mode
- Fix the fallback image for the 'Me' button
- Remove the site header in the 'Reader' when selecting a site from 'Filter'
- Code & improvements in `ReaderTabViewModel`, introduced `ReaderTabItemStore`

To test:

- Go to 'My Site' and verify that the fallback image corresponds to `userCircle` in `GridIcons`
  - **Note**: usually you'll need an account with no gravatar for this check, but at the moment there's a bug (whose fix is not yet merged into develop) that prevents to load the gravatar image, so you can use any account.

- Enable Dark Mode, go to the 'Reader' and check that the styles of all the dividers are consistent between themselves and with other dividers in the app (e.g. the tab bar in 'Stats')

- From the reader, tap 'Filter' and select a site, verify that no header is displayed. Clear the filter, and select a site in a different way (e.g. from 'Search' or tapping on the site icon/name in a card), and verify that the header shows up.

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ will be in a eparate PR.
- [x] ~~I have considered adding accessibility improvements for my changes.~~ will be in a eparate PR.
